hera/editor/panels/DesignPanel.tsx

```diff
@@ -255,6 +256,51 @@ export default function DesignPanel({
   return (
     <Tick animationConfig={AnimationConfig}>
       <Stack gap={24} vertical verticalPadding>
+        <Box center gap={32}>
+          {/* TODO(dkratz): Remove text and add selected state to buttons */}
```

Comment:
You can either use `selected` or `selectedText` on `InlineLink`. They are different active states for links. You can use the editor's drawing mode to compare the state.

Reply:
Done: bdf6ab5df7bd46f53f073a06ac64b66d5c66bcbd